numpy/distutils/system_info.py

```diff
@@ -362,6 +362,11 @@ def add_system_root(library_root):
 so_ext = get_shared_lib_extension()
 
 
+def is_symlink_to_accelerate(filename):
+    return (sys.platform == 'darwin' and os.path.islink(filename) and
+            'Accelerate' in os.path.realpath(filename))
```

Comment:
Could this be `os.path.realpath(filename).startswith('/System/...whatever.../Accelerate.framework')`, to be more specific, given that the installation location probably is always the same?

Reply:
OK, I'll make it more specific.

Reply:
Fixed.